wasp-ai/src/client/pages/MainPage.jsx

```diff
@@ -78,7 +78,7 @@ const MainPage = () => {
         localStorage.setItem("appNum", 0);
       }
       if (appNum === 2) {
```

Comment:
`appNum` is not the best name -> maybe `numGenerationsSoFar` instead? Or `numGenerationsStarted`?